src/plugins/cosmos/CosmosAssetAccountDetails.tsx

```diff
@@ -22,7 +23,11 @@ type AssetDetailsProps = {
 }
 
 export const CosmosAssetAccountDetails = ({ assetId, accountId }: AssetDetailsProps) => {
+  const isOsmosisSwapEnabled = useFeatureFlag('OsmosisSwap')
   const assetIds = useMemo(() => [assetId], [assetId])
+  const defaultBuyAssetId =
```

Comment:
do you mind putting this in the same spot as all the other logic?

https://github.com/shapeshift/web/blob/release/src/components/Trade/hooks/useSwapper/utils.ts#L197

Reply:
sure thing c21949ef943423d69e7f62bedd425aebd8ca03d7